meinberlin/assets/scss/components_user_facing/_card.scss

```diff
@@ -33,4 +33,39 @@
 
 .card__pill {
     margin-bottom: 0.3em;
+}
+
+.card__footer .functions {
+    display: flex;
+    order: 1;
+    align-self: flex-end;
+    margin-top: -9px;
+    margin-right: -18px;
+}
+
+.card__button--shortlist:before,
+.card__button--answered:before {
+    background-color: $primary;
+}
+
+.card__button i {
+    font-size: 14px;
+}
+
+.card__button--hidden i,
+.card__button--live i {
+    color: $gray;
+}
+
+.card__button--shortlist i,
+.card__button--answered i {
+    color: $white;
```

Comment:
You can use `$text-inverted` here. https://github.com/liqd/a4-meinberlin/blob/3b0db77db26f1ed06e6b2340cd35a39c3cee90f0/meinberlin/assets/scss/styles_user_facing/variables/_colors.scss#L22